### PR TITLE
rel to #10140: remove html and cdata from strings.xml resources

### PR DESCRIPTION
**Pull request by eddiemuc**
rel to #10140: remove html and cdata from strings.xml resources

### CODE REVIEW

File: main/res/values-el/strings.xml [hunk 1]
```diff
@@ -1869,7 +1869,7 @@
     <string name="receivemapfile_cancelled">Η μεταφορά του αρχείου χάρτη ακυρώθηκε</string>
     <!-- Download map preference and menu entry -->
     <string name="downloadmap_title">Λήψη χαρτών εκτός σύνδεσης</string>
-    <string name="downloadmap_info">Επιλέξτε ένα αρχείο χάρτη για την περιοχή σας πατώντας το εικονίδιο \"Αποθήκευση\".\n\n 
+    <string name="downloadmap_info">Επιλέξτε ένα αρχείο χάρτη για την περιοχή σας πατώντας το εικονίδιο \"Αποθήκευση\".\n\n
```
Comment (Lineflyer) [inline main/res/values-el/strings.xml:1872]: This looks unrelated.

Comment (Lineflyer) [reply main/res/values-el/strings.xml:1872]: but correct, so lets merge

Comment (eddiemuc) [reply main/res/values-el/strings.xml:1872]: Maybe my android Studio inserted a whitespace somewhere? Can't really see what happened here
